components/pagination/collections/paging_collection.js

```diff
@@ -377,6 +373,9 @@
                 }
             }
         });
+
+        var PagingCollectionProto = PagingCollection.prototype;
```

Comment:
I definitely find it confusing to define it below the lines that are using it. If only we could use ES2015's ``let``!